can/interfaces/canalystii.py

```diff
@@ -69,132 +57,355 @@ class VCI_CAN_OBJ(Structure):
     logger.info("Cannot load CANalystII library")
 
 
+class VCI_INIT_CONFIG(Structure):
+    _fields_ = [
+        ("AccCode", c_ulong),
+        ("AccMask", c_ulong),
+        ("Reserved", c_ulong),
+        ("Filter", c_ubyte),
+        ("Timing0", c_ubyte),
+        ("Timing1", c_ubyte),
+        ("Mode", c_ubyte),
+    ]
+
+
+class VCI_CAN_OBJ(Structure):
+    _fields_ = [
+        ("ID", c_uint),
+        ("TimeStamp", c_int),
+        ("TimeFlag", c_byte),
+        ("SendType", c_byte),
+        ("RemoteFlag", c_byte),
+        ("ExternFlag", c_byte),
+        ("DataLen", c_byte),
+        ("Data", c_byte * 8),
+        ("Reserved", c_byte * 3),
+    ]
+
+
+class VCI_BOARD_INFO(Structure):
+    _fields_ = [
+        ('hw_Version', c_ushort),
+        ('fw_Version', c_ushort),
+        ('dr_Version', c_ushort),
+        ('in_Version', c_ushort),
+        ('irq_Num', c_ushort),
+        ('can_Num', c_byte),
+        ('str_Serial_Num', c_char * 20),
+        ('str_hw_Type', c_char * 40),
+        ('Reserved', c_ushort * 4)
+    ]
+
+
+class VCI_CAN_STATUS(Structure):
+    _fields_ = [
+        ('ErrInterrupt', c_ubyte),
+        ('regMode', c_ubyte),
+        ('regStatus', c_ubyte),
+        ('regALCapture', c_ubyte),
+        ('regECCapture', c_ubyte),
+        ('regEWLimit', c_ubyte),
+        ('regRECounter', c_ubyte),
+        ('regTECounter', c_ubyte),
+        ('Reserved', c_ulong)
+    ]
+
+
+class VCI_ERR_INFO(Structure):
+    _fields_ = [
+        ('ErrCode;', c_uint),
+        ('Passive_ErrData', c_byte * 3),
+        ('ArLost_ErrData', c_byte)
+    ]
+
+
+_in_use = {}
+
+
 class CANalystIIBus(BusABC):
     def __init__(
         self,
-        channel,
-        device=0,
-        bitrate=None,
-        Timing0=None,
-        Timing1=None,
+        channel: int,
+        device: int = 0,
+        bitrate: Optional[int] = None,
+        Timing0: Optional[int] = None,
+        Timing1: Optional[int] = None,
         can_filters=None,
         **kwargs,
     ):
         """
+        CANalystII Interface
+
+        :param channel: channel number to use
+        :type channel: int
```

Comment:
I don't think you need to specify the type here since you added types to the interface.

Reply:
Does sphinx pick up on the data types when compiling the documentation if the type is not supplied in the docstring? I was not sure of the answer to that question so I simply included the type in the docstring thinking that it wouldn't hurt at all.